src/worldmap_path.c

```diff
@@ -1865,9 +1865,9 @@ int GetPlayChapterId(u32 chapterId)
     int i;
 
     if (chapterId - 0x25 < 9)
```

Comment:
It may be worthwhile for us to consider converting this into a macro for `CHAPTER_IS_TOWER` (and similarly below, `CHAPTER_IS_RUINS`), but not necessary for this PR